packages/desktop-client/package.json

```diff
@@ -87,6 +87,7 @@
     "start": "cross-env PORT=3001 node scripts/start.js",
     "start:browser": "./bin/watch-browser",
     "watch": "cross-env PORT=3001 node scripts/start.js",
+    "locales:validate": "node scripts/validate-locales.js",
```

Comment:
This is a good idea, I've added something similar to this in 5c56370920926177b2f53aa33c9309ac25cd8845 after I saw this.

`i18next-parser` looks for all the translation keys which exist in the codebase and checks to make sure that they exist in the locales files. This is a stronger check as it'll pick up missing keys even if none of the locales files have it.

Reply:
Yes, i saw that. I will revert my changes and let only `check-i18n` that looks better.